include/snowflake/client.h

```diff
@@ -403,6 +419,11 @@ typedef struct SF_TIMESTAMP {
     SF_DB_TYPE ts_type;
 } SF_TIMESTAMP;
 
+/**
+ * Initializes an SF_QUERY_RESPONSE_CAPTURE struct.
+ */
+SF_QUERY_RESULT_CAPTURE snowflake_query_result_capture_init();
+
 /**
  * Checks whether the client is running in force_arrow mode.
  *
```

Comment:
I missed this in the last review, but can we prefix this with `snowflake` and add STDCALL the the function definition